api/storage/client.go

```diff
@@ -180,7 +180,7 @@ func (c *Client) Attach(unitId string, storageIds []string) ([]params.ErrorResul
 }
 
 // Destroy destroys specified storage entities.
-func (c *Client) Destroy(storageIds []string, destroyAttached bool) ([]params.ErrorResult, error) {
+func (c *Client) Destroy(storageIds []string, destroyAttachments, releaseStorage bool) ([]params.ErrorResult, error) {
```

Comment:
I was all ready to change this to Remove, but I'm hesitating now because it makes it different to all the other places we use the name Destroy in the client APIs. Probably better to keep consistency, and put "underlying" in the arg name somewhere.